app/src/main/java/com/minimalisticapps/priceconverter/domain/usecase/GetCoinsUseCase.kt

```diff
@@ -26,26 +29,18 @@ constructor(
                 it.value.code = it.key.uppercase()
                 val rate = it.value.rate
                 if (rate != null) {
-                    val oneShitCoinValue = 1.div(rate)
-                    it.value.oneShitCoinValue = oneShitCoinValue
-                    it.value.oneShitCoinValueString = when {
```

Comment:
This should NOT be here. This is formatting issue and it should be responsibility of a given component NOT the data model!